apps/www/src/components/paynote-overlay/paynote-overlay.tsx

```diff
@@ -343,10 +343,31 @@ function PaynoteOverlayDialog() {
   )
 }
 
-export function PaynoteOverlay() {
+function isPaynoteOverlayHidden(
+  pathname: string,
+  searchParams: URLSearchParams,
+): boolean {
   return (
+    (pathname === '/angebote' && searchParams.has('package')) ||
+    pathname === '/mitteilung' ||
+    pathname === '/anmelden' ||
+    pathname.startsWith('/konto') ||
+    pathname === '/meine-republik' ||
+    pathname === '/community' ||
+    searchParams.has('extract') ||
+    searchParams.has('extractId')
```

Comment:
would it make sense to have the list as an env variable? or at least a const?

Reply:
i thought about it but env doesn't make much sense because changing it would require a rebuild anyway. a const just adds more indirection because we differentiate between startsWith, query params, pathname …